index.html

```diff
@@ -3,11 +3,11 @@
 <head>
 	<title>Exercise: Gradient Background Generator</title>
 	<link rel="stylesheet" type="text/css" href="./style.css">
-	<script type="text/javascript" src="./script.js" defer></script>
+	<script type="text/javascript" src="./bundle.js" defer></script>
```

Comment:
bundle.js is not uploaded into GitHub, and I could come fix this later on. This is for demonstration and educational purposes, since I'm learning how to use GitHub